helm-charts/splunk-otel-collector/values.yaml

```diff
@@ -1179,6 +1179,9 @@ operator:
       nodejs:
         repository: ghcr.io/signalfx/splunk-otel-js/splunk-otel-js
         tag: v2.6.0
+      dotnet:
+        repository: ghcr.io/signalfx/splunk-otel-dotnet/splunk-otel-dotnet
+        tag: 1.2.0
```

Comment:
Depends on https://github.com/signalfx/splunk-otel-dotnet/pull/397 - the tag should be `1.2.1`